src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,14 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+import './UserInfo.scss';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer